packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -891,6 +891,35 @@ export default class Meeting extends StatelessWebexPlugin {
         });
       }
 
```

Comment:
@taymoork2  do you want to add the then block for updateShare and then do the stop as it might cause start then stop local

Reply:
1) check localStopped is triggering before remote start , ? not. add then block to updatShare and add start remoteL logic there

2) In a 3 party call user A receiving the share events , Remote Share start from user B and Stopped remote share user B then start Remote SHare user C ? add the check for previous and current content ID